docs/develop/dialog-api-in-office-add-ins.md

```diff
@@ -206,6 +211,39 @@ function processMessage(arg) {
 > [!NOTE]
 > The `showNotification` implementation is not shown in the sample code provided by this article. For an example of how you might implement this function within your add-in, see [Office Add-in Dialog API Example](https://github.com/OfficeDev/Office-Add-in-Dialog-API-Simple-Example).
 
+### Cross-domain messaging to the host runtime
+
+The dialog may be navigated away from the add-in's domain or the parent JavaScript runtime (either in a task pane or a UI-less runtime that hosts a function file) may be navigated away from the add-in's domain after the dialog is opened. If either of these things has happened, then a call of `messageParent` will fail unless your code specifies the domain of the parent runtime. You do this by adding a [DialogMessageOptions](/javascript/api/office/office.dialogmessageoptions) parameter to the call of `messageParent`. This object has a `targetOrigin` property that specifies the domain to which the message should be sent. If the parameter is not used, Office assumes that the target is the same domain that the dialog is currently hosting.
+
+> [!NOTE]
+> Using `messageParent` to send a cross-domain message requires the [Dialog Origin 1.1 requirement set](../reference/requirement-sets/dialog-origin-requirement-sets.md).
+
+The following is an example of using `messageParent` to send a cross-domain message.
+
+```js
+Office.context.ui.messageParent("Some message", { targetOrigin: "https://target.domain.com" });
+```
+
+> [!NOTE]
+> The `DialogMessageOptions` parameter was released approximately July 19th, 2021. For about 30 days after that date, in Office on the web, the first time that `messageParent` is called without the `DialogMessageOptions` parameter and the parent is a different domain from the dialog, the user will be prompted to approve sending data to the target domain. If the user approves, the user's answer is cached for 24 hours. The user is not prompted again during this period when `messageParent` is called with the same target domain.
+
+If the message doesn't include sensitive data, you can set the `targetOrigin` to "\*" which allows it to be sent to any domain. The following is an example.
+
+```js
+Office.context.ui.messageParent("Some message", { targetOrigin: "*" });
+```
+
+> [!TIP]
```

Comment:
"The simplest way to do this" -> "To do this, create a file..."

Reply:
I guess if this is a breaking change I would maybe expect this to be something like a warning and not a tip?

Reply:
Don't know if you need "In the meantime"

Reply:
Do you need to specify Windows computer? Reason I point this out is that I am unsure how recent announcements of Win365 Cloud PC affect processes like this running virtually but on a Mac.

Reply:
It's not a breaking change in the usual sense. This is just an optional thing users/admins can do. So, I think is should be a tip.

I think it is understood in the developer world that a virtual machine (e.g. virtualized Windows running in a Mac) counts as a Windows computer.  If it wasn't, then all the world's documentation that mentions an OS-specific thing, would have to be followed by a parenthetical note saying something like "or a virtualized instance running in another host OS".